Octokit/Models/Request/TeamRepositoryUpdate.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Octokit.Internal;
+using System.Diagnostics;
+using System.Globalization;
+
+namespace Octokit
+{
+    public enum PermissionType
```

Comment:
There already exists a `RepositoryPermission` enum that has the same values so I think you could just use that